### PR TITLE
Initial port of JMS20 FAT from WS-CD-Open

### PR DESCRIPTION
**Pull request by tbitonti**
Initial step of porting the JMS20 FATs from WS-CD-Open.  This initial push contains all of the test content, but is enabled only to run LiteBucketSet1Test and LiteBucketSet2Test.

### CODE REVIEW

**Comment by tbitonti** [comment]
This pull request is to deliver FAT tests.  No deliverable code changes are made.

**Comment by tbitonti** [comment]
#build

**Comment by tbitonti** [comment]
Will be updating copyrights.

**Comment by dhaggart** [comment]
Based on the build failure:

 dev/com.ibm.ws.messaging.open_jms20_fat/.project

 -> update project name

 dev/com.ibm.ws.messaging.open_jms20_fat/build-test.xml 

 -> update project name

   However, in my ports (e.g. *.open_clientcontainer_fat) I got rid of the Ant build script; everything being done with gradle typically for OpenLiberty.

   (I had expected it to be ignored but I guess if it is there then it is used...)

File: dev/com.ibm.ws.messaging.open_jms20_fat/build-test.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dazavala) [inline dev/com.ibm.ws.messaging.open_jms20_fat/build-test.xml:1]: I don't see where this part is used.  If this part is now vestigial, consider removing it when the test suite is fully migrated.  The file serves only to confuse those who maintain the feature.

File: dev/com.ibm.ws.messaging.open_jms20_fat/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#*******************************************************************************
```
Comment (dazavala) [inline dev/com.ibm.ws.messaging.open_jms20_fat/bnd.bnd:1]: Update the copyright here.  Also, I noticed this copyright inconsistency in a few other files.

**Comment by tbitonti** [comment]
For https://github.com/OpenLiberty/open-liberty/issues/11851

**Comment by tbitonti** [comment]
#libby

**Comment by tbitonti** [comment]
#libby

**Comment by tbitonti** [comment]
#build

**Pull request merged**